src/styles/main.scss

```diff
@@ -1,3 +1,56 @@
-body {
-  margin: 0;
+.main {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+  font-family: Arial, sans-serif;
+  margin: 9px;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  margin: auto;
+  width: 721px;
+  padding: 10px;
+  gap: 1px;
+
+  &__field {
```

Comment:
Use variables for field dimensions (width, height) and also for calendar gap. With this, you will be able to calculate everything else